client/src/components/Ticketing/cart/Cart.tsx

```diff
@@ -140,15 +145,15 @@ const Cart = (): ReactElement => {
     setRemoveContext(RemoveContext.single);
     setRemoveContextMessage('this');
     setTargetItem({eventInstanceId: eventInstanceId, ticketTypeId: ticketTypeId});
-    handleClick2();
+    openModal();
   };
 
   const navigateToCompleteOrder = () => {
     history('/completeorder');
   };
 
   return (
-    <main className='flex flex-col items-center h-full py-[5rem] px-[1rem] tab:px-[5rem] bg-zinc-200'>
+    <main className='flex flex-col items-center min-h-[calc(100vh-233px)] md:min-h-[calc(100vh-142px)] h-full py-[5rem] px-[1rem] tab:px-[5rem] bg-zinc-200'>
```

Comment:
I found that on larger displays, i.e 1440p, the footer would be rendered in the center of the screen on the cart, so these classes make the min-height of the cart screen, the screen size minus the height of the footer.